### PR TITLE
by primsi: Pass additional parameters to widgets.

### PR DESCRIPTION
**Pull request by primsi**

### CODE REVIEW

File: src/Entity/EntityBrowser.php [hunk 1]
```diff
@@ -189,6 +196,21 @@ protected function displayPluginCollection() {
   }
 
   /**
+   * {@inheritdoc}
+   */
+  public function setAdditionalWidgetParameters(array $parameters) {
```
Comment (slashrsm) [inline src/Entity/EntityBrowser.php]: addAdditional...

File: src/Plugin/EntityBrowser/Widget/View.php [hunk 2]
```diff
@@ -45,8 +45,17 @@ public function getForm(array &$original_form, FormStateInterface $form_state) {
         ->getExecutable();
     }
 
-    $form['view'] = $storage['view']->executeDisplay($this->configuration['view_display']);
+    if (!empty($this->configuration['arguments'])) {
+      if (!empty($aditional_widget_parameters['path_parts'])) {
+        $arguments = array_intersect_key($aditional_widget_parameters['path_parts'], array_flip($this->configuration['arguments']));
```
Comment (slashrsm) [inline src/Plugin/EntityBrowser/Widget/View.php:51]: 'arguments' should be added to config schema

File: src/Plugin/EntityBrowser/Widget/View.php [hunk 3]
```diff
@@ -45,8 +45,17 @@ public function getForm(array &$original_form, FormStateInterface $form_state) {
...
+        $storage['view']->setArguments(array_values($arguments));
+      }
+      else {
```
Comment (slashrsm) [inline src/Plugin/EntityBrowser/Widget/View.php]: Probably not. Is it ok if we execute view w/o argument. That case is handled on view itself, right?

Comment (primsi) [inline src/Plugin/EntityBrowser/Widget/View.php]: Yes.

File: config/schema/entity_browser.schema.yml [hunk 4]
```diff
@@ -82,3 +82,8 @@ entity_browser.browser.widget.view:
     view_display:
       type: string
       label: 'View display ID'
+    arguments:
+      type: sequence
+      sequence:
+        - type: integer
+        - label: 'View display arguments'
```
Comment (slashrsm) [inline config/schema/entity_browser.schema.yml]: I believe it need to be 

```
arguments:
  type: sequence
  sequence:
    - type: integer
      label: 'View display arguments'
```

(w/o the second -)

**Pull request merged**